FreeTAKServer/controllers/configuration/MainConfig.py

```diff
@@ -28,282 +28,181 @@ class MainConfig:
-
-            websocketkey = str(os.environ.get('FTS_WEBSOCKET_KEY', yamlConfig["Certs"].get("FTS_WEBSOCKET_KEY", "YourWebsocketKey")))
-
-            CRLFile = str(os.environ.get('FTS_CRLDIR', yamlConfig["Certs"].get("FTS_CRLDIR", fr"{certsPath}/FTS_CRL.json")))
-        else:
-            federationKeyPassword = str(os.environ.get('FTS_FED_PASSWORD', 'defaultpass'))
-
-            keyDir = str(os.environ.get("FTS_SERVER_KEYDIR", Path(fr'{certsPath}/server.key')))
-
-            pemDir = str(os.environ.get("FTS_SERVER_PEMDIR", Path(fr'{certsPath}/server.pem')))  # or crt
-
-            testPem = str(os.environ.get("FTS_TESTCLIENT_PEMDIR", pemDir))
-
-            testKey = str(os.environ.get("FTS_TESTCLIENT_KEYDIR", keyDir))
-
-            unencryptedKey = str(os.environ.get("FTS_UNENCRYPTED_KEYDIR", Path(fr'{certsPath}/server.key.unencrypted')))
-
-            p12Dir = str(os.environ.get("FTS_SERVER_P12DIR", Path(fr'{certsPath}/server.p12')))
-
-            CA = str(os.environ.get("FTS_CADIR", Path(fr'{certsPath}/ca.pem')))
-
-            CAkey = str(os.environ.get("FTS_CAKEYDIR", Path(fr'{certsPath}/ca.key')))
-
-            federationCert = str(os.environ.get("FTS_FEDERATION_CERTDIR", Path(fr'{certsPath}/server.pem')))
-
-            federationKey = str(os.environ.get("FTS_FEDERATION_KEYDIR", Path(fr'{certsPath}/server.key')))
-
-            federationKeyPassword = str(os.environ.get("FTS_FEDERATION_KEYPASS", 'defaultpass'))
-
-            password = str(os.environ.get('FTS_CLIENT_CERT_PASSWORD', 'supersecret'))
-
-            websocketkey = str(os.environ.get('FTS_WEBSOCKET_KEY', "YourWebsocketKey"))
-
-            CRLFile = str(os.environ.get('FTS_CRLDIR', fr"{certsPath}/FTS_CRL.json"))
+    # User Connection package IP needs to be set to the IP which is used when creating the connection in your tak device
+    UserConnectionIP = ip
 
+    # API IP and port
+    APIIP = '0.0.0.0'
+    APIPort = 19023
 
+    # Federation port
+    FederationPort = 9000
 
     # allowed ip's to access CLI commands
     AllowedCLIIPs = ['127.0.0.1']
 
     # IP for CLI to access
     CLIIP = '127.0.0.1'
 
+    ## API Settings ##
     APIVersion = "1.9.5"
+    nodeID = f"FreeTAKServer-{uuid4()}"
+    OptimizeAPI = True
+    DataReceptionBuffer = 1024
+    MaxReceptionTime = 4
+
+    # number of milliseconds to wait between each iteration of main loop
+    # decreasing will increase CPU usage and server performance
+    # increasing will decrease CPU usage and server performance
+    MainLoopDelay = 100
+
+    # set to None if you don't want a message sent
+    ConnectionMessage = f'Welcome to FreeTAKServer {version}. The Parrot is not dead. It’s just resting'
+
+    ## Database Settings ##
+    # whether or not to save CoT's to the DB
+    SaveCoTToDB = True
+
+    # this should be set before startup
+    DBFilePath = '/opt/FTSDataBase.db'
+    DataBaseType = 'SQLite'
```

Comment:
maybe it would make sense to add a note specifying that MySql is not supported at the moment

Reply:
I can add the comment. Not a problem. 

Correct me if I am wrong FTS only support SQLite and PostgreSQL, right?

Reply:
I was thinking about this and I am sure that there is something way down in the code that is preventing this, but FTS is using SqlAlchemy if I remember correctly and it seems that any of (or at least most of the common ones) that SqlAlchemy supports would work for storing FTS state. What is stopping FTS from supporting more databases?